### ISSUE TITLE
[7.0.0+]The collapsibleColumns and autoRowSize break the table visibility when loaded ( [8.0.0+] inside an iframe)

### ISSUE THREAD

**Issue by AMBudnik** [handsontable/handsontable#7424]
### Description

The `collapsibleColumns` and `autoRowSize` break the table visibility when loaded inside an iframe.

### Steps to reproduce

1. Open the demo to see that table does not load.

7.4.2
![image](https://user-images.githubusercontent.com/10757813/101461719-958f9200-393b-11eb-895f-9d6b0bd6abe7.png)

8.0.0
![image](https://user-images.githubusercontent.com/10757813/101461811-b22bca00-393b-11eb-9892-d0d260c0f2f9.png)

### Demo

* 8.0.0 (broken) - https://jsfiddle.net/f3jo0d5b/
* 7.4.2 (works well) - https://jsfiddle.net/sh0kfreg/

### Your environment
* Handsontable version: 8+
* Browser Name and version: Chrome 87
* Operating System: Windows 10

**Comment by AMBudnik** [comment]
### Case 2

It seems that when we skip the iframe the issue is still and it does not work for v7 as well.

* 6.0.0 (works well) - https://jsfiddle.net/gkn9zdxL/1/ 
* 6.2.2 (works well) - https://jsfiddle.net/gkn9zdxL/6/
* 7.0.0 (broken) - https://jsfiddle.net/gkn9zdxL/2/ 
* 8.0.0 (broken) - https://jsfiddle.net/gkn9zdxL/4/

**Comment by swistach** [comment]
In fact, it throws an error at initialization:
```
Uncaught TypeError: Cannot read property 'parentNode' of null
    at CollapsibleColumns.onAfterGetColHeader (handsontable.full.js:113505)
    at Core.<anonymous> (handsontable.full.js:113198)
    at Hooks.run (handsontable.full.js:7546)
    at Core.runHooks (handsontable.full.js:55272)
    at TableView.appendColHeader (handsontable.full.js:74217)
    at GhostTable.createColumnHeadersRow (handsontable.full.js:40427)
    at GhostTable.addColumnHeadersRow (handsontable.full.js:40222)
    at AutoRowSize.calculateRowsHeight (handsontable.full.js:82925)
    at AutoRowSize.onBeforeRender (handsontable.full.js:83227)
    at Core.<anonymous> (handsontable.full.js:82864)
```

**Comment by aninde** [comment]
This issue has been fixed by v8.3.2.
Both cases work correctly. CollapsibleColumns works with autoRowSize.

Case 1 with the iframe https://jsfiddle.net/zhLk8wmp/ (fixed).
Case 2 no iframe https://jsfiddle.net/1czq5036/ (fixed).

**Comment by AMBudnik** [comment]
Thank you for testing both cases. We can close this issue as solved in v8.3.2.
